content/tutorials/quickstart/py-create-time-based-escrows.md

```diff
@@ -0,0 +1,639 @@
+
+To create a time-based escrow:
+
+1. Enter an **Amount** to transfer. For example, _100000000_.
+2. Copy the **Operational Account** value.
+3. Paste it in the **Destination Account** field.
+4. Set the **Escrow Finish (seconds)** value. For example, enter _10_.
+5. Set the **Escrow Cancel (seconds)** value. For example, enter _120_.
+6. Click **Create Time-based Escrow**.
+7. Copy the _Sequence Number_ of the escrow called out in the **Standby Result** field.
+
+The escrow is created on the XRP Ledger instance, reserving 100 XRP plus the transaction cost. When you create an escrow, capture and save the **Sequence Number** so that you can use it to finish the escrow transaction.
+
+[![Completed Create Escrow Transaction](img/quickstart-py-escrow3.png)](img/quickstart-py-escrow3.png)
+
+## Finish Escrow
+
+The recipient of the XRP held in escrow can finish the transaction any time within the time window after the Escrow Finish date and time but before the Escrow Cancel date and time. Following on the example above, you can use the _Sequence Number_ to finish the transaction once 10 seconds have passed.
+
+To finish a time-based escrow:
+
+1. Paste the sequence number in the Operational account **Sequence Number** field.
+2. Click **Finish Escrow**.
+3. Click **Get Op Account Info** and **Get Standby Account Info** to update their **XRP Balance**.
+
+The transaction completes and balances are updated for both the Standby and Operational accounts.
+
+[![Completed Escrow Transaction](img/quickstart-py-escrow4.png)](img/quickstart-py-escrow4.png)
+
+## Get Escrows
+
+Click **Get Escrows** to see the current list of escrows for the Operational account. If you click the buttons now, there are no escrows at the moment.
+
+For the purposes of this tutorial, follow the steps in [Create Escrow](#create-escrow), above, to create a new escrow transaction. Remember to capture the _Sequence Number_ from the transaction results.
+
+Click **Get Escrows**. The `account_info` request returns the same `account_object` for both accounts, demonstrating the link between the accounts created by the escrow transaction.
+
+[![Get Escrows results](img/quickstart-py-escrow5.png)](img/quickstart-py-escrow5.png)
+
+
+## Cancel Escrow
+
+When the Escrow Cancel time passes, the escrow is no longer available to the recipient. The initiator of the escrow can reclaim the XRP, less the transaction fees. If you try to cancel the transaction prior to the **Escrow Cancel** time, you are charged for the transaction, but the actual escrow cannot be cancelled until the time limit is reached.
```

Comment:
It feels odd to call out less the transaction fees as the full escrowed amount is returned the transaction fee is paid by the escrow cancel not subtracted from the balance of the escrow.

Reply:
Makes sense. I'll take out that clause.